cmake/variables.cmake

```diff
@@ -18,8 +18,11 @@ set(SNAPSHOT yes)
 
 set(ML_APP_NAME controller)
 
-execute_process(COMMAND awk -F= "/elasticsearchVersion/ {gsub(/-.*/,\"\"); print $2}"
-  $ENV{CPP_SRC_HOME}/gradle.properties OUTPUT_VARIABLE ML_VERSION_NUM OUTPUT_STRIP_TRAILING_WHITESPACE)
+file(READ ${CMAKE_SOURCE_DIR}/gradle.properties GRADLE_PROPERTIES)
+if(${GRADLE_PROPERTIES} MATCHES "elasticsearchVersion=([0-9\.]+)\n")
```

Comment:
Does this need to explicitly match the newline character?  I am thinking it would be better if it didn't.  As it stands it will break if somebody introduces a trailing space into the Gradle properties file.

Reply:
Good point! I'll make the change.